### PR TITLE
Update maintainers

### PR DESCRIPTION
**Pull request by gdevenyi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,12 +14,12 @@ or run `make` in this directory for a list of helpful commands.
 Maintainers:
 
 * [Gabriel Devenyi][devenyi_gabriel]
-* [Ashwin Srinath][srinath_ashwin]
 * [Colin Morris][colin_morris]
 * [Will Pitchers][will_pitchers]
+* [Gerard Capes][[gerard_capes]
```
Comment (gcapes) [inline README.md]: Typo: Double open square bracket :(

File: README.md [hunk 2]
```diff
@@ -14,12 +14,12 @@ or run `make` in this directory for a list of helpful commands.
...
 
 [devenyi_gabriel]: http://software-carpentry.org/team/#devenyi_gabriel
-[srinath_ashwin]: http://software-carpentry.org/team/#srinath_ashwin
 [colin_morris]: https://github.com/colinmorris
 [will_pitchers]: https://software-carpentry.org/team/#pitchers_w
+[gerard_capes]: http://colinmorris.github.io/about
```
Comment (gcapes) [inline README.md]: Incorrect link :( 
The existing ones only take you to the list of instructors, but let's use this for now: https://carpentries.org/instructors/#capes_gerard

Comment (gdevenyi) [reply README.md]: Oops, there's a broken copy paste on my part :-/

File: README.md (hunk 1 above)
Comment (gdevenyi) [reply README.md]: And there's a failure of my highligher :-/

Comment (gdevenyi) [reply README.md]: I really need to wean myself off of nano....

**Comment by gcapes** [comment]
:) Great, cheers!

**Pull request merged**